### PR TITLE
Sign Out button added and tested

### PR DESCRIPTION
**Pull request by walkonmars36**

### CODE REVIEW

File: src/containers/Profile/Profile.jsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
-import React from 'react'
+import React from "react";
+import "./Profile.scss";
+import Button from "../../components/Button/Button";
```
Comment (Charlie-robin) [inline src/containers/Profile/Profile.jsx:3]: We want the button to be able to sign the user out -> https://firebase.google.com/docs/auth/web/password-auth#next_steps

This won't need to be tested 👍

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

File: src/containers/Profile/Profile.jsx (hunk 1 above)
Comment (walkonmars36) [reply src/containers/Profile/Profile.jsx:3]: Sign out from firebase no in place

File: src/containers/Profile/Profile.jsx [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+import {auth} from "../../firebase";
+import {signOut} from "firebase/auth";
 
 // path: /profile
 
 const Profile = () => {
+  const logOut = () => {
+    signOut(auth)
+      .then(() => {
```
Comment (Charlie-robin) [inline src/containers/Profile/Profile.jsx:12]: You don't need these `.then()` they don't do anything of use.

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

**Review by CarapaceonBear — APPROVED** [review]
looks good to me!

File: src/containers/Profile/Profile.jsx (hunk 2 above)
Comment (HPTR) [reply src/containers/Profile/Profile.jsx]: Removed!

**Pull request merged**